src/base/_responsive-prop.scss

```diff
@@ -0,0 +1,20 @@
+@mixin responsive-prop($property, $mobile, $value: false) {
+
+  @include small-only {
```

Comment:
Does this work as is? Don't you need to import 'ambience/responsive'?

Reply:
Oh, you import it every time you use this. This is not smart. Import it here, so you don't need to re-import everywhere.

Reply:
Ok